components/style/properties/properties.mako.rs

```diff
@@ -1091,6 +1098,10 @@ impl PropertyDeclaration {
                     match input.try(|i| CSSWideKeyword::parse(context, i)) {
                         Ok(CSSWideKeyword::InheritKeyword) => {
                             % for sub_property in shorthand.sub_properties:
+                                if properties_seen.get_${property.ident}() {
```

Comment:
> I read the property.ident value outside the property loop, and yet the property is still available. Is mako supposed to allow this behaviour?

Mako compiles to Python, where a variable’s scope is an entire function or method. So after exiting a `for x in xs:` loop, the last `x` value is still in scope.